app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class UserDisabled < StandardError
 
 class User < ActiveRecord::Base
   rolify
+  has_many :physical_tickets, through: :ticket_purchases, dependent: :destroy
```

Comment:
you need the `join`. But anyway the by_conference method is always used like:

`user.physical_tickets.by_conference`

so you can just add the `by_conference` scope here, and even remove it from the other model, as would only be used here.